.github/WorkingAgreement.md

```diff
@@ -50,99 +50,13 @@ Our crews are distributed across timezones and across the world. Where possible,
-- Acceptance Criteria is defined
-- User stories are achievable withinin the milestone
-  - Stories are broken down into prioritized tasks ranging from small to large (If extra large, break it down)
-  - "Spike" if investigating something in order to timebox and track outcomes
-- Story owner is able to break down user story into tasks if desired
-- Dependencies identified (either external or other work items)
-
-#### Definition of Done
-
-- Acceptance Criteria are satisfied
-- Appropriate [Pull Request template(s)](https://github.com/retaildevcrews/ngsa/blob/main/.github/PULL_REQUEST_TEMPLATE.md) satisfied
-- [Pull Request](https://github.com/orgs/retaildevcrews/projects/4) approved and completed
-- [DoD Review & Release](https://github.com/orgs/retaildevcrews/projects/4?card_filter_query=label%3Arelease) checklist satisfied and completed
-- Demonstration recorded and available to customer (when applicable)
-
-### Backlogs and (Dash)boards
+### Pairing
 
-- Product Owner owns the Product Backlog.
-- Tech Lead owns the Risk Backlog.
-- User Story Board will be used to track User Story progress.
-- Kanban Board will be used to track project progress
-  - Board columns:
-    - **Triage**: All net-new tasks/bugs/features/stories need to be created as an "issue"; things to discuss/notes can be added as a "note"
-    - **Backlog**: Stories and tasks that have been refined, triaged, and prioritized.
-    - **Sprint Goals**: Stories that have been committed for the current sprint.
-    - **Sprint Backlog**:  Tasks that have been committed for the current sprint.
-    - **In Progress**: A development team member owns the story or bug and begins work.
-    - **PR Submitted/In Review**: The owner of the story or bug determines the item meets our Definition of Done and has created a Pull Request. The item will stay in this status through the PR process -- including addressing requested feedback or fixing issues found.
-    - **Done**: The Pull Request/Task has completed, and the work has been committed to the `main` branch of the project repository.
+Pairing work is recommended to support knowledge sharing between the team members. Work items have a dedicated field *Pairing with* so that a second team members is explicitly assigned. Each team member should look for pairing opportunities before picking up a new work item. Commits should be made by the people that are assigned to the story. Other team members are encouraged to provide feedback using PR comments.
 
-### Estimating
+### Sharing
 
-- We will estimate User Stories with size tagging to help gauge how much work we commit to within a sprint.
-- We will use T-shirts sizing for our estimation -- with XS <1 day, Small is 1-2 days, Medium is 2-3 days, Large is 4-5 days, and XL > week, needs decomposed.
+Sharing is caring. We will record demos to share early and share often. We will share as we go and not wait until the end of the project.
 
 ### Ceremonies
 
```

Comment:
Do we want to rotate Scrum Master weekly?

Reply:
Not sure on that, that could be a good discussion for the offsite meeting though.